tests/python/pants_test/backend/python/rules/test_python_fmt_integration.py

```diff
@@ -1,36 +1,76 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
+from contextlib import contextmanager
 from os.path import relpath
 from pathlib import Path
 
-from pants.util.contextutil import temporary_file, temporary_file_path
-from pants_test.pants_run_integration_test import PantsRunIntegrationTest, ensure_daemon
+from pants.util.contextutil import temporary_dir, temporary_file_path
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+def write_build_file(root_dir):
+  build = Path(root_dir, "BUILD")
+  build.touch
+  build.write_text("python_library()")
+
+INCONSISTENTLY_FORMATTED_HELLO: str = "def hello():x=42"
+CONSISTENTLY_FORMATTED_HELLO: str = "def hello():\n    x = 42\n"
+
+
+def write_inconsistently_formatted_file(root_dir, filename) -> Path:
```

Comment:
Consider either `pants.util.contextutil.temporary_file` or `pants.util.dirutil.safe_file_dump`

Reply:
> Wherever you deleted imports, could you please check if the BUILD entries are correct for those targets too? For example, removing `pants.util.objects` imports where possible.

Done. I only found one case. Is there a way to do this automatically with pants? (if I could introduce one, there may be many in the codebase)